distribution/docs/src/main/resources/_contents/_plugins/metacardingest-network-contents.adoc

```diff
@@ -0,0 +1,75 @@
+==== Metacard Ingest Network Plugin
```

Comment:
Should probably put a `-` in the file name between `metacard` and `ingest` for consistency. 

Also, I didn't see that this section was getting included in the docs anywhere.